### DISCUSSION TITLE
我切换小号扫码登录后，切换大号 服务就LOG OUT了

### DISCUSSION THREAD

**Discussion by Ancss** [zhayujie/chatgpt-on-wechat#559] [Q&A] [answered]
我用大号切换小号，桌面端微信也退出了登录
执行 python3 app.py，用小号扫码进行登录，如果切换回大号，服务就打印LOG OUT!
正常是这样的吗，这样我岂不是大号小号永远无法进行对话

**Comment by zhayujie** [comment, marked as answer]
正常是的哦，因为微信的设定是 手机端退出，其他所有端同时退出。

你可以和别人对话， 或者Android手机可以双开

**Reply by Ancss** [reply]
知道了
